plugins/udp/udp_output.go

```diff
@@ -38,12 +43,16 @@ type UdpOutputConfig struct {
 	Address string
 	// Optional address to use as the local address for the connection.
 	LocalAddress string `toml:"local_address"`
+
+	// Maximum size of message, plugin drops the data if it exceeds this limit.
+	MaxMessageSize int `toml:"max_message_size"`
```

Comment:
Should validate the user provided configuration value and have a lower threshold (512 should be reasonable).